### DISCUSSION TITLE
Browse detections in chronological order

### DISCUSSION THREAD

**Discussion by HansNorelius** [Mattk70/Chirpity-Electron#143] [Q&A] [answered]
How do I get back to the initial results after browsing on a particular bird speices? 

I often dive in to the diferent bird speices detected and check the  most interesting results. Sometimnes I would like to go back to the initial chronological order of all detections but I don't know how to do that. Any tips?
MvH Hasse

**Comment by Mattk70** [comment, marked as answer]
If you click on a species in the summary table on the left to filter the results to just that species, click on the species again to remove that species filter.
